dev_scripts/repro-build

```diff
@@ -0,0 +1,665 @@
+#!/usr/bin/env python3
+
```

Comment:
All the other files in this folder have `.py` suffix. Should we stick with it?

Reply:
Done in 094d876dbaae6ce51654283bbcde8609e239b887.